content/1_General/Running_Code_Locally.mdx

```diff
@@ -274,6 +274,50 @@ work with the latter two. This is one of the reasons why you should **not** use
 If you install GCC as described later in the module linked above, then you
 should be good to go.
 
+### Precompiling `<bits/stdc++.h>`
+
+<Resources>
+	<Resource
+		source="CF"
+		title="Speed up GCC Compile Time"
+		url="https://codeforces.com/blog/entry/53909"
+	 />
+</Resources>
+
+Using this trick speeds up compilation time to less than a second!
+
+Make sure to compile `stdc++.h` with the flags that you normally use to compile your C++ program.
+
+#### Option 1 (Mentioned in CF blog)
+
+```
+mkdir bits
+cp /usr/include/path_to/stdc++.h bits/
+cd bits
+g++ stdc++.h
+```
+
+Then, change `#include <bits/stdc++.h>` to `#include "bits/stdc++.h"`. Changing this tells the compiler to look for the local version.
+
+#### Option 2
+
+```
+g++ /usr/include/path_to/bits/stdc++.h
```

Comment:
For me, I had to use this with flag -H in order to get it to work, not sure about others.